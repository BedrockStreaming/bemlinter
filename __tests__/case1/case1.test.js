const bemlinter = require('./../../src/bemlinter');

it('renders correctly', (done) => {
  bemlinter(`${__dirname}/*.scss`)
    .then(logs => {
      expect(logs).toMatchSnapshot();
      done();
    });
});