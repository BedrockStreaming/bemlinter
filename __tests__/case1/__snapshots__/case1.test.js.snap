exports[`test renders correctly 1`] = `
Object {
  "alright": Object {
    "info": Array [
      Object {
        "filePath": "./__tests__/case1/alright.scss",
        "line": null,
        "message": "Linting finish",
      },
    ],
  },
}
`;
